README.md

```diff
@@ -32,6 +32,14 @@ bindsym $mod+p exec i3_quickterm
 # always pop standard shell, without the menu
 bindsym $mod+b exec i3_quickterm shell
 ```
+Building From Source
+--------------------
+
+```
```

Comment:
As the package has been uploaded on PyPI recently, the recommended installation method should probably be

```
pip3 install i3-quickterm
```

In practice, you might need sudo or `--user` but I think we can left that out, as it just uses the most common python conventions.

People cloning the repository by hand will probably want to change the code, so I expect they won't need the instructions to clone the repository.